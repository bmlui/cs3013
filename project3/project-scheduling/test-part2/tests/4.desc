analysis: SJF: with one job (and for parsing newline...)
