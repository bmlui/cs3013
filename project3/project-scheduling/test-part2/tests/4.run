./scheduler 1 SJF tests/4.in 0
