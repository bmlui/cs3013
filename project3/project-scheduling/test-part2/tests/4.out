Execution trace with SJF:
t=0: [Job 0] arrived at [0], ran for: [5]
End of execution with SJF.
Begin analyzing SJF:
Job 0 -- Response time: 0  Turnaround: 5  Wait: 0
Average -- Response: 0.00  Turnaround 5.00  Wait 0.00
End analyzing SJF.
